src/looper.ts

```diff
@@ -102,6 +103,18 @@ export function isIDValid(config: Config, ids: string): boolean {
   return true
 }
 
+export function loopCheckSTUNServer(interval: number) {
+  const checkSTUNinterval = setInterval(async () => {
```

Comment:
can we also add a condition if interval is < = 0, no need to loop the checkSTUNinterval

Reply:
no loop, but just check once to get public ip